media/css/base/banners/turning-red.scss

```diff
@@ -0,0 +1,201 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+$brand-theme: 'firefox';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+$color-panda-red: #fc4055;
+
+#turning-red-banner {
+    &.c-banner {
+        @include clearfix;
+        @include at2x('/media/img/banners/turningred/mobile.jpg', 320px, 110px);
+        background-color: $color-panda-red;
+        background-position: center top;
+        background-repeat: no-repeat;
+        border-bottom: 1px solid rgba(0, 0, 0, 0.25) inset;
+        color: $color-white;
+        min-width: $content-xs;
+        padding-top: 110px;
+        position: relative;
+        text-align: center;
+        z-index: 3;
+
+        .mzp-l-content {
+            padding: $spacing-lg;
+        }
+
+        // hide by default if JS is available to avoid flicker
+        // (if visitor previously dismissed)
+        .js & {
+            display: none;
+        }
+
+        // conditional class used to display the banner.
+        &.c-banner-is-visible {
+            display: block;
+        }
+    }
+
+    .c-banner-main {
+        max-width: $content-sm;
+        margin: 0 auto;
+    }
+
+    .c-button {
+        @include image-replaced;
+        background: transparent url('/media/img/banners/turningred/button-fun.png') center top no-repeat;
```

Comment:
Issue (blocking): locally this button is invisible for me. It looks like it's using a different source image to the one on demo?

![image](https://user-images.githubusercontent.com/400117/154682601-c4220b10-3650-44fb-b42c-bae84d125e1d.png)

Reply:
Oops, I forgot to commit the latest version of the button after I made it a sprite for hover (and the reason this is an old-school image sprite is because I don't want to add the whole font for one button, and a PNG ended up being smaller than SVG).